R/rope.R

```diff
@@ -325,33 +337,77 @@ rope.stanfit <- rope.stanreg
 
 #' @rdname rope
 #' @export
-rope.brmsfit <- function(x, range = "default", ci = .89, ci_method = "HDI", effects = c("fixed", "random", "all"), component = c("conditional", "zi", "zero_inflated", "all"), parameters = NULL, verbose = TRUE, ...) {
+rope.brmsfit <- function(
+  x, 
+  range = "default", 
+  ci = .89, 
+  ci_method = "HDI", 
+  effects = c("fixed", "random", "all"), 
+  component = c("conditional", "zi", "zero_inflated", "all"), 
+  parameters = NULL, 
+  verbose = TRUE, 
+  ...
+) {
   effects <- match.arg(effects)
   component <- match.arg(component)
 
-  if (insight::is_multivariate(x)) {
-    stop("Multivariate response models are not yet supported.")
-  }
-
+  # check range argument
   if (all(range == "default")) {
     range <- rope_range(x)
+  # we expect a list with named vectors (length two) in the multivariate case.
+  # Names state the response variable.
+  } else if (insight::is_multivariate(x) && 
+    (!is.list(range) || length(range) != length(insight::find_response(x) ||
+        names(range) != insight::find_response(x)))) {
```

Comment:
Not sure if the parenthesis are set correctly here?

I guess it should be

```r
(!is.list(range) || length(range) != length(insight::find_response(x)) || names(range) != insight::find_response(x))
```

Reply:
Good catch, thank you!